.prow.yaml

```diff
@@ -147,6 +147,8 @@ presubmits:
           value: "1.16.15"
         - name: TEST_SET
           value: "conformance"
+        - name: TF_VAR_ami
+          value: ami-08985edfecbbbcf52
```

Comment:
Why are we using the custom AMI and what's exactly that AMI?

Reply:
This is ubuntu 18.04 image. The one that will be selected by default by machine-controller.

Reply:
```
$ curl -s https://uec-images.ubuntu.com/query/bionic/server/released.current.txt | grep ami-08985edfecbbbcf52
bionic	server	release	20210119.1	ebs-ssd	amd64	eu-west-3	ami-08985edfecbbbcf52			hvm
```

Reply:
For older kubernetes versions older docker (if docker is chosen as a runtime) should be used, older docker presents only in `bionic` docker repository.